### ISSUE TITLE
plugin breaks association between label and input

### ISSUE THREAD

**Issue by koelnconcert** [shaack/bootstrap-input-spinner#87]
According to [WCAG](https://www.w3.org/WAI/tutorials/forms/labels/) a html `label` element should be used and associated with the corresponding `input` element - either explicitly via a `for` and `id` attribute or implicitly via nesting of the elements.

Both approaches fail with this plugin, because the original (hidden) input field is still the first input element (relevant for nesting) or the original `for`-`id`-association is unchanged, see my [jsfiddle example](https://jsfiddle.net/koelnconcert/jmn1v23d).

**Comment by koelnconcert** [comment]
One possible solution would be to generate a new id for the new input element and adjust all label-for-attributes existing for the original id.

This would at least solve the problem for explicit associations.

**Comment by shaack** [comment]
See example "Destroying the spinner" here: https://shaack.com/projekte/bootstrap-input-spinner/
